cmd/cronjob-checker/main.go

```diff
@@ -81,12 +90,11 @@ func main() {
 		lastRunTimeV1 := cronGet.Status.LastScheduleTime
 		lastRunTime := lastRunTimeV1.Time
 		shouldOfRun := findLastCronRunTime(schedule)
-		timeMinus := shouldOfRun.Add(time.Duration(-30) * time.Second)
-		timePlus := shouldOfRun.Add(time.Duration(30) * time.Second)
+		tM, tP := scheduleWindow(shouldOfRun, time.Minute*5)
 
 		log.Infoln("last run time for cronjob", c.Name, "was", lastRunTime)
 
-		if lastRunTime.Before(timeMinus) && lastRunTime.Before(timePlus) {
+		if lastRunTime.Before(tM) && lastRunTime.Before(tP) {
```

Comment:
```suggestion
		if lastRunTime.After(tM) && lastRunTime.Before(tP) {
```

What we want to say here is: "if the last run time was after the `earliestAllowedTime` and the last run time was before the `latestAllowedTime`.

Reply:
done. changed logic to make this the positive